anonymizer/OpenDiffix.Service/CSVFormatter.fs

```diff
@@ -0,0 +1,38 @@
+module OpenDiffix.Service.CSVFormatter
+
+open OpenDiffix.Core
+open OpenDiffix.Core.QueryEngine
+
+let private quoteString (string: string) =
+  "\"" + string.Replace("\"", "\"\"") + "\""
+
+let private csvFormat value =
+  match value with
+  | String string -> quoteString string
+  | _ -> Value.toString value
+
+let format suppressedAnonCount (result: QueryResult) =
```

Comment:
It's bad abstraction to be aware of `suppressedAnonCount` here. The star bucket row can be prepended to the row sequence at `O(1)` if we allow for a `seq<Row>` input.